spdx/parsers/tagvalue.py

```diff
@@ -1287,6 +1301,14 @@ def p_snippet_lic_comment_1(self, p):
         msg = ERROR_MESSAGES["SNIP_LICS_COMMENT_VALUE"].format(p.lineno(1))
         self.logger.log(msg)
 
+    def p_text_or_line_value_1(self, p):
+        """text_or_line : TEXT"""
+        p[0] = p[1]
+
+    def p_text_or_line_value_2(self, p):
+        """text_or_line : LINE"""
+        p[0] = p[1]
```

Comment:
This looks absurd, but seems to be doing the right thing!

Reply:
Yes, in the tag-value lexer, a `TEXT` is a string (possibly multi-line) enclosed in `<text>`, while `LINE` is a single line without the tags. Some properties don't allow `TEXT`, so there has to be a distinction.